specification/compute/resource-manager/readme.md

```diff
@@ -829,6 +829,30 @@ input-file:
 - Microsoft.ContainerService/preview/2015-11-01-preview/containerService.json
 ```
 
```

Comment:
Hi @ansahdev did you intentionally add tag description at the bottom? The tag descriptions are listed in descending order and this should be at the top.

Reply:
Moved the new tags to the top of the list.